src/components/AuthorSelect/AuthorSelect.css

```diff
@@ -2,7 +2,9 @@ ul.author-select {
   list-style-type: none;
   padding: 0;
   margin: 0;
-  font-size: 17px; 
+  font-size: 0.75rem;
+  display: flex;
+  flex-flow: row wrap;
 }
```

Comment:
The only (nit-picky) thing I'm noticing is the way that `flex-flow: row wrap` will cause the labels to become mis-aligned. IMO a grid arrangement would look cleaner better. But with CSS grid, we can't define a grid without pre-setting the number of columns. Which means it's tricky on different devices. I.e.: it's tricky.

We could perhaps set a grid and switch between four and two columns, using different screen width breakpoints...

But in any case: that's just the icing on the cake.

Reply:
P.S: alternatively, and if we want to save even more space on the map views, we could turn it into a drop-down menu (like the nav menu). But as I said: icing on the cake :-)

Reply:
I agree with that, and was considering a grid as well! I think 3 columns might work on all non-desktop displays. I think I will merge this and once we get your PR #52 merged, I'll open a new one with revisions to the mobile display, since it is impacted by that PR.